go/ct/st/code_test.go

```diff
@@ -105,22 +104,6 @@ func TestCode_GetData(t *testing.T) {
 	}
 }
 
-func TestCode_CopyTo(t *testing.T) {
```

Comment:
Although it is very simple there should still be a unit test for the Code Copy function.

Reply:
I do not think this issue was addressed as requested.